src/jstests/formio/components/fixtures/phonenumber.js

```diff
@@ -0,0 +1,15 @@
+const phoneNumberComponent = {
+  label: 'Phone Number',
+  key: 'phonenumber',
+  type: 'phoneNumber',
+  input: true,
+  validate: {
+    backendApi: true,
+    plugins: ['phonenumber-international', 'phonenumber-nl'],
+  },
+};
+
+const validSamples = ['630123456', '+31630123456'];
```

Comment:
```suggestion
const validSamples = ['0630123456', '+31630123456'];
```
Doesn't the first one need a 0?